modules/exploits/windows/browser/ovftool_format_string.rb

```diff
@@ -90,15 +90,15 @@ def ovf
 xmlns:cim="http://schemas.dmtf.org/wbem/wscim/1/common"
 xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1"
 xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData"
-xmlns:vmw="http://www.vmware.com/schema/ovf"
+xmlns:vmw="https://www.vmware.com/schema/ovf"
 xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <References>
     <File ovf:href="Small VM-disk1.vmdk" ovf:id="file1" ovf:size="68096" />
   </References>
   <DiskSection>
     <Info>Virtual disk information</Info>
-    <Disk ovf:capacity="8" ovf:capacityAllocationUnits="#{fs}" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" />
+    <Disk ovf:capacity="8" ovf:capacityAllocationUnits="#{fs}" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="https://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" />
```

Comment:
What happened here? Were these `http -> https` changes to exploit code intended?

Reply:
nope, that was not, thus the dangers of a blind string replace. good catch.